charm/docs/explanation/charm-architecture.md

```diff
@@ -0,0 +1,29 @@
+# NetBox charm architecture
+
+The NetBox charm initial version has been generated with the 
+help of the paas-app-charmer project. The paas-app-charmer project provides many
+of the functionalities needed by this charm like:
+- PostgreSQL integration
+- Django migrations
+- Ingress integration
+- COS (Prometheus metrics and Loki logs for gunicorn).
```

Comment:
Doesn't it include a dashboard too?

Reply:
Not yet, we are still discussing how to add specific metrics/log files with `paas-app-charmer`, and at that point at a proper dashboard (otherwise, it would be better is `paas-app-charmer` provides it).

Reply:
Gotcha!